src/locales/en/en.json

```diff
@@ -142,6 +142,36 @@
   "COMMON_USERS":"User",
   "COMMON_YES":"Yes",
   "COMMON_NO":"No",
+  "LABEL_HOME":"Home",
+  
+	"ADD_AUTH": "Add Auth Token",
```

Comment:
fix indents, remove white spaces at the beginning of all lines / or match those with rest of the lines